.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,44 @@
-**Description:**
+## First
+- [ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation.
```

Comment:
This is a bit intimidating, isn't it? :smile: 
My approach would rather be to Write it out at the start, with no heading, with a suggestion to delete this sentence at the end

```
[ ] I have read and understand the [Contributing](https://github.com/home-assistant/home-assistant/CONTRIBUTING.md) documentation. Note that you may delete this line.
```